.gitignore

```diff
@@ -0,0 +1,20 @@
+# Compiled Object files
```

Comment:
This was added by mistake, but maybe it's good to have it in by default in the repo?